templates/server.yaml.j2

```diff
@@ -111,10 +117,16 @@ spec:
           configMap:
             name: "kadalu-info"
         - name: glusterfsd-datadir
-{% if brick_device == "" %}
+{%- if brick_device == "" %}
           hostPath:
             path: "{{ host_brick_path }}"
             type: Directory
-{% else %}
+{%- else %}
           emptyDir: {}
-{% endif %}
+{%- endif %}
+{%- if brick_device_dir != "" %}
```

Comment:
Looks like we need to get the similar logic for 'path' too. Otherwise, path directory is not exported inside glusterfsd container IMO.

Reply:
It is already handled for path directory https://github.com/aravindavk/kadalu/blob/master/templates/server.yaml.j2#L114 

Looks like not working as expected, I will debug further.

Reply:
Silly mistake... used `or` instead of `and` here https://github.com/aravindavk/kadalu/blob/master/server/glusterfsd.py#L117

Reply:
Cool :+1